src/stores/ui.ts

```diff
@@ -1,9 +1,12 @@
 import { defineStore } from 'pinia'
 
 import type { OpenverseCookieState, SnackbarState } from '~/types/cookies'
-import { isProd } from '~/utils/node-env'
 
-import type { CookieSerializeOptions } from 'cookie'
+import { cookieOptions } from '~/types/cookies'
+
+import type { Locale } from 'vue-i18n'
+
+export type BannerId = Locale | 'cc-referral'
```

Comment:
The translation banner ids are actually `` `translation-${Locale}` `` not just the locale string.